fixtures/loginFixture.ts

```diff
@@ -0,0 +1,19 @@
+import { test as base } from '@playwright/test';
+import dotenv from '.env.example';
+
+dotenv.config();
+
+const username = process.env.USERNAME;
+const password = process.env.PASSWORD;
+
+export const test = base.extend({
+    loginPage: async ({ page }, use) => {
```

Comment:
login method should be in POM page, and here in fixtures you import this page, and use it